crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -220,6 +224,52 @@ impl GlobalTransform {
         }
     }
 
+    /// Divide `self` with `transform`, this is the reciprocal of [`Self::mul_transform`].
+    ///
+    /// `t2 * t1 / t2 == t1` Note that transforms are not commutative, meaning that
```

Comment:
This is confusing because the divisor is on the left, but it applies the inverse on the left. This will cause a lot of confusion. Furthermore, there should be two division operators, but `Div` can't represent this.

Reply:
I'll be honest and admit I'm a bit confused. I've tried making sense of it on paper and my understanding is that **if an inverse existed on the domain of transforms** then it would be identical to `(t2 * t1) * t2^-1`. Am I wrong? How is that confusing? It does mirror the real numbers division (ie: `a / b` == `a * b^-1`)

Also why should there be two division operations?

Reply:
Personally (not speaking for @HackerFoo) I would not define an inverse like this.

I'd instead define it given an operation `(*, 1)`, then an inverse for a given `x`, `y` would be such that `x * y = 1`. Then `/` can be defined as `*` by the inverse. Hence I'd expect `a * (b / b) = a * 1 = a`. 

For a well-formed `(*, 1)`, of course `a * (b / b) = (a * b) / b = (b * a) / b` discontinuities non-withstanding which is the relationship that you have put.

However, the substitution steps steps above rely on the fact that `(*, 1)` is both commutative and associative and the precedence rules between `*` and `/` don't matter. Even without commutativity and associativity I'd very much expect that `t1 * (t2 / t2) == t1 * 1` should hold (where `1` is the identity matrix).

Finally, I'd expect a second division operation for scalar division of matrices.

Reply:
Thank you for the explanation. I see. So maybe it would be better to have a different name, for example "difference" or "unparent"? This way, not only it doesn't lead to confusion, but it hints at potential usage.